### DISCUSSION TITLE
For once you google guys must do something that actually works

### DISCUSSION THREAD

**Discussion by sidoncloud** [GoogleCloudPlatform/generative-ai#146] [General]
AttributeError: 'TextGenerationModel' object has no attribute 'tune_model' while running model.tune_model() .

**Comment by sidoncloud** [comment]
For anyone who is not able to proceed , use the below import statement . These guys can seem to neither update nor change their source code for others to use or reference them 

`from vertexai.preview.language_models import TextGenerationModel`

instead of 

`from vertexai.language_models import TextGenerationModel`

**Comment by polong-lin** [comment]
Hi @sidoncloud ,

Thank you very much for spotting this issue and providing the error message.

Here's the fix that you can use for now before the PR is merged:
https://github.com/GoogleCloudPlatform/generative-ai/pull/147/commits/8e09522e9a48ee0cbd2ffa41c2dec5716d5c176b

PR:
https://github.com/GoogleCloudPlatform/generative-ai/pull/147
